src/components/pages/datatable/create/form/index.tsx

```diff
@@ -86,8 +86,11 @@ export default function DataTableCreateForm({
           )
           .required()
           .test({
-            message: 'Scientific name or Common name mapping is required',
-            test: (value) => value?.some((item) => ['scientificName', 'commonName'].includes(item?.fieldKey || "")) || false
+            message: "Scientific name or Common name mapping is required",
+            test: (value) =>
+              value?.some((item) =>
+                ["scientificName", "commonName"].includes(item?.fieldKey || "")
```

Comment:
I think we should be just fine without `|| ""`

![image](https://user-images.githubusercontent.com/5774849/150351518-2616626d-5fdc-4df1-a785-5c2034a821ad.png)